lib/logger.js

```diff
@@ -1,15 +1,15 @@
 const winston = require('winston')
 const options = require('./options')
+const _ = require('lodash')
 
 const logger = new winston.Logger({
   level: options.logLevel,
   transports: [
     new (winston.transports.Console)({ timestamp: true, colorize: true })
   ],
-  rewriters: [(_, __, meta) => {
-    if (meta) return meta.toString()
-  }]
-
+  rewriters: [
+    (...[,, meta]) => _.hasIn('toString', meta) ? meta.toString() : meta
```

Comment:
Hmm, if meta doesn't have a ``toString`` function, we probably don't want to print it out. Maybe this should just be some fixed message, like "Error, no further information is available".

Reply:
Good catch! What a brain fart =p